ci/scripts/e2e-iceberg-sink-v2-test.sh

```diff
@@ -52,6 +53,8 @@ poetry run python main.py -t ./test_case/iceberg_source_all_delete.toml
 poetry run python main.py -t ./test_case/iceberg_source_explain_for_delete.toml
 poetry run python main.py -t ./test_case/iceberg_predicate_pushdown.toml
 
+echo "--- Running benchmarks"
+poetry run python main.py -t ./test_case/bench_predicate_pushdown.toml
```

Comment:
We should avoid manually listing all test cases here. Can we use glob?

Reply:
If we want to distinguish bench, better to put it in a different directory, just like `e2e_test/slow_tests`

Reply:
Done, thanks